themes/startwords/layouts/_default/baseof.html

```diff
@@ -60,7 +60,7 @@
 
     <!-- alternative formats -->
     {{ range .AlternativeOutputFormats -}}
-    <link rel="{{ .Rel }}" type="{{ .MediaType.Type }}" href="{{ .Permalink | safeURL }}">
+    {{ printf `<link rel="%s" type="%s" href="%s" />` .Rel .MediaType.Type .Permalink | safeHTML }}
```

Comment:
nice, I was hoping to do something cool like this! `.MediaType.Type` is awesome.

Reply:
oh wait, we can't do this for articles - can we?

Reply:
@thatbudakguy not sure what you mean about articles. This is the base template so we should get this behavior there too. I would expect it to pick up the text but not the pdf (since it isn't an alternate that Hugo knows about / manages).

Reply:
sorry, I'm mixing up `<head>` metadata with the actual output format download links in the article body. was mostly wondering if there's anything going on here that we could use in the same way to generate the output format links in a "smarter" way (autodetect MIME type?), but I don't think we can.

Reply:
@thatbudakguy we're already using output formats similar to this to get the txt link for the article body. I think we could get the mime type from the object instead of hard coding it, but I'm not sure how much that gets us. (It would make more sense if we could do that for both pdf and text.)